### PR TITLE
Strong name everything.

### PR DESCRIPTION
**Pull request by cesarbs**
https://github.com/aspnet/dnx/issues/2615

cc @davidfowl @muratg @Eilon @pranavkm @moozzyk @Tratcher

### CODE REVIEW

**Comment by pranavkm** [comment]
All of these are shared src files. There shouldn't be binaries in here to sign

**Comment by Tratcher** [comment]
Not all of them

**Comment by cesarbs** [comment]
@pranavkm But I see binaries in the packages built from those projects. Should I ignore that?

**Comment by pranavkm** [comment]
Do they get referenced by the application? If not, skip it for the shared src projects.

**Comment by cesarbs** [comment]
Updated. Not signing *.Sources.

File: src/Microsoft.Extensions.MemoryPool/project.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "version": "1.0.0-*",
+  "compilationOptions": {
+    "keyFile": "../../tools/Key.snk"
```
Comment (pranavkm) [inline src/Microsoft.Extensions.MemoryPool/project.json]: Unrelated, but could you throw in a `warningAsErrors` here?

**Comment by pranavkm** [comment]
:shipit:

**Pull request merged**